packages/Webkul/Velocity/src/Resources/views/shop/home/new-products.blade.php

```diff
@@ -1,13 +1,14 @@
 @php
     $count = core()->getConfigData('catalog.products.homepage.no_of_new_product_homepage') ?: 10;
-
+    $guest_review_status = core()->getConfigData('catalog.products.review.guest_review');
     $direction = core()->getCurrentLocale()->direction == 'rtl' ? 'rtl' : 'ltr';
 @endphp
 
 {!! view_render_event('bagisto.shop.new-products.before') !!}
 
 <product-collections
     count="{{ (int) $count }}"
+    :guest-review-status="{{ (Boolean) $guest_review_status ? 'true' : 'false'}}"
```

Comment:
Be consistent when type casting if above you using a small case then below should also be small case like bool or boolean.